src/pages/PersonalDetails.jsx

```diff
@@ -40,6 +42,8 @@ const PersonalDetails = () => {
     const onSubmit = async (data) => {
         setLoading(true);
         const formData = new FormData();
+        formData.append('firstName', data.bio);
```

Comment:
fix line number 45 to 46 
change data.bio to data.firstName likewise do with lastName

Reply:
okey